aws/offsite/aurora-snapshots-fargate/README.md

```diff
@@ -0,0 +1,22 @@
+Runs a script which verifies our offiste database backups once per day.
+
+## Deployment
+
+1. Build the docker image:
+`docker build -t <your tag> .`
+
+2. Push image to ECR repository named `aurora-verify-backups`, tagged with `latest`
```

Comment:
Oh yeah, I forgot there would be a few extra manual steps needed here for building + pushing the custom Docker image containing the code to the repository, versus building + pushing the Lambda zip-package to S3.

We might want to think about how to improve/automate this process later if we want to duplicate this process for similar tasks in the future, but this works well enough for now.

Reply:
Yea, for this one in particular it's tricky because it needs to be in the offsite account. In general I could imagine either 1). Running automated docker builds and pushes from staging, or 2). using codebuild / drone to do so. For #2 it would be easier if these were in their own repository.